code/datums/migrants/waves/bodyguard.dm

```diff
@@ -20,7 +20,7 @@
 	mask = /obj/item/clothing/face/spectacles/inqglasses
 	beltr = /obj/item/weapon/knife/cleaver/combat
 	belt = /obj/item/storage/belt/leather/steel
-	backr = /obj/item/storage/backpack/satchel
+	wear_back_right = /obj/item/storage/backpack/satchel
```

Comment:
this clashes with the others now and makes beltr/l inconsistent, but I don't expect all of them to be changed to wear_